sklearn/neighbors/_kde.py

```diff
@@ -72,6 +72,11 @@ class KernelDensity(BaseEstimator):
         metric.  For more information, see the documentation of
         :class:`BallTree` or :class:`KDTree`.
 
+    Attributes
+    ----------
+    tree_ : BinaryTree
```

Comment:
should it be a BallTree or a KDTree?

in any case could you do the following change

```suggestion
    tree_ : ``BinaryTree`` instance
```